### PR TITLE
Resorts back-end

### PR DESCRIPTION
**Pull request by jakxnz**
https://trello.com/c/kapm0rle

### CODE REVIEW

File: plugins/underflip/resorts/models/Rating.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    /**
+     * @var string
+     */
+    public $filterColumn = 'value';
+
+    /**
+     * @var array
+     */
+    protected $validOperators = [
+        '=', '<', '>', '<=', '>=', '<>', '!=',
```
Comment (Krank3n) [inline plugins/underflip/resorts/models/Rating.php:32]: interesting

File: plugins/underflip/resorts/updates/seed_units_table.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        Unit::create([
+            'name' => 'percentage',
+            'title' => 'Percentage',
+            'singular_title' => 'Percent',
+            'plural_title' => 'Percent',
+            'format' => '%s%',
+        ]);
+
+        Unit::create([
```
Comment (Krank3n) [inline plugins/underflip/resorts/updates/seed_units_table.php:36]: I see some places its called ratings and others score? Will this be an issue?

**Review by Krank3n — APPROVED** [review]

File: plugins/underflip/resorts/Plugin.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    /**
+     * @return array
+     */
+    public function pluginDetails()
+    {
+        return [
+            'name' => 'Resorts',
+            'description' => 'Provides the Resorts part of Shred Index',
+            'author' => 'Underflip',
+            'icon' => 'icon-icon-snowflake-o'
```
[the remark this answers is not in the document]
Comment (jakxnz) [reply plugins/underflip/resorts/Plugin.php:20]: It's true, none of the icons are particularly good. I'll leave it as it for now, but it'll be cool if we can enhance it with something more intuitive later

File: plugins/underflip/resorts/updates/seed_units_table.php (hunk 2 above)
Comment (jakxnz) [reply plugins/underflip/resorts/updates/seed_units_table.php:36]: As per our chat on Slack. I think I'll leave it as-is for now, but happy to have these improved as we go along.

**Comment by jakxnz** [comment]
[Resorts back-end](https://trello.com/c/kapm0rle/14-resorts-back-end)

**Pull request merged**